test/controller/auth.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 const supertest = require('supertest-as-promised');
-const expect = require('chai').expect;
+const expect = require('chai').use(require('dirty-chai')).expect;
```

Comment:
One caveat about `no-unused-expressions` is that chai allows expressions like this:

```js
expect(foo).to.be.true;
```

This works because it has a getter on the "true" property, which throws if `foo` isn't `true`. However, ESLint doesn't realize this, so it flags that pattern as an unused expression.

The solution is to use a chai plugin called [`dirty-chai`](https://github.com/prodatakey/dirty-chai), which replaces those getters with functions. So the above assertion would be written as:

```js
expect(foo).to.be.true();
```